functionary/ui/templates/socialaccount/login.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% block title %}
+    Sign In
+{% endblock title %}
+{% block content %}
+    <div class="content">
+        {% if process == "connect" %}
+            <h1>Connect {{ provider.name }}</h1>
+            <p>You are about to connect a new third party account from {{ provider.name }}.</p>
+        {% else %}
+            <h1>Sign In via {{ provider.name }}</h1>
+            <p>You are about to sign in using a third party account from {{ provider.name }}.</p>
```

Comment:
In the sign-in scenario this ends up looking a little jarring because it builds off of the base.html.  So you go from the clean login form over to a page with the Functionary banner for the redirect.  It's the kind of thing that we can smooth out later if we want, but I figured mention it.

Reply:
Added a copy of Jace's modal to handle this.

There's some magic happening behind the scenes when you're not logged in to your gitlab account, like the token's not valid and it forces a different code path. The result is that you will always get prompted to confirm logging in via third party, vs just when you're not currently signed in to the thirdpary.

I tried getting around this by using the seemingly HTTP calls that the confirm uses(POST vs GET to the same URL), but I was getting CORS errors.

Reply:
The behavior as you have it now seems good to me.